src/core/components/states/stateSkeletonBar/index.ts

```diff
@@ -0,0 +1,5 @@
+export {
+    StateSkeletonBar,
+    type IStateSkeletonBarProps,
+    type SkeletonBarSize as SkeletonSize,
```

Comment:
I'd suggest to use the component name as prefix and remove the alias to avoid confusions
```suggestion
    type StateSkeletonBarSize,
```

Reply:
Oof, this is a vsCode rename. Good catch

Reply:
@Fabricevladimir wdyt about the renaming from `SkeletonBarSize` to `StateSkeletonBarSize`

Reply:
It's coming 👍🏾